tidy3d/components/tcad/data/monitor_data/charge.py

```diff
@@ -0,0 +1,220 @@
+    )
+
+    electrons: HeatChargeDataset = pd.Field(
+        None, title="Electrons series", description="Contains the electrons."
+    )
+
+    holes: HeatChargeDataset = pd.Field(
+        None, title="Holes series", description="Contains the electrons."
+    )
+
+    @pd.root_validator(skip_on_failure=True)
+    def warn_no_data(cls, values):
+        """Warn if no data provided."""
+
+        mnt = values.get("monitor")
+        electrons = values.get("electrons")
+        holes = values.get("holes")
+
+        if electrons is None or holes is None:
+            log.warning(
+                f"No data is available for monitor '{mnt.name}'. This is typically caused by "
+                "monitor not intersecting any solid medium."
+            )
+
+        return values
+
+    @property
+    def symmetry_expanded_copy(self) -> StaticChargeCarrierData:
+        """Return copy of self with symmetry applied."""
+
+        new_electrons = self._symmetry_expanded_copy(property=self.electrons.field_series)
+        new_holes = self._symmetry_expanded_copy(property=self.holes.field_series)
+
+        return self.updated_copy(
+            electrons_series=self.electrons.updated_copy(field_series=new_electrons),
+            holes_series=self.holes.updated_copy(field_series=new_holes),
+        )
+
+    def field_name(self, val: str) -> str:
+        """Gets the name of the fields to be plot."""
+        if val == "abs^2":
+            return "Electrons², Holes²"
+        else:
+            return "Electrons, Holes"
+
+
+class StaticCapacitanceData(HeatChargeMonitorData):
+    """Class that stores capacitance data from a Charge simulation."""
+
+    monitor: StaticCapacitanceMonitor = pd.Field(
+        ...,
+        title="Capacitance monitor",
+        description="Capacitance data associated with a Charge simulation.",
+    )
+
+    hole_capacitance: ElectroStaticDataArray = pd.Field(
+        None,
+        title="Hole capacitance",
+        description="Small signal capacitance (dQh/dV) associated to the monitor.",
+    )
+
+    electron_capacitance: ElectroStaticDataArray = pd.Field(
+        None,
+        title="Electron capacitance",
+        description="Small signal capacitance (dQe/dV) associated to the monitor.",
+    )
+
+    @pd.validator("hole_capacitance", always=True)
+    @skip_if_fields_missing(["monitor"])
+    def warn_no_data(cls, val, values):
+        """Warn if no data provided."""
+
+        mnt = values.get("monitor")
+
+        if val is None:
+            log.warning(
+                f"No data is available for monitor '{mnt.name}'. This is typically caused by "
+                "monitor not intersecting any solid medium."
+            )
+
+        return val
+
+    def field_name(self, val: str) -> str:
+        """Gets the name of the fields to be plot."""
+        return ""
+
+
+class StaticVoltageData(HeatChargeMonitorData):
+    """Data associated with a :class:`VoltageMonitor`: spatial electric potential field.
```

Comment:
What about `ElectrostaticPotentialData`? Also, with minimum modifications this one and the free carrier one can be reused for harmonic response so they would not longer be DC or static. Having said that, maybe derived classes could also be defined